Tests/Build-Bicep.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+try {
+    $ScriptDirectory = Split-Path -Path $PSCommandPath -Parent
+    Import-Module -FullyQualifiedName "$ScriptDirectory\..\Source\Bicep.psd1"
+}
+catch {
+    Throw "Unable to import Bicep module. $_"
+}
+
+InModuleScope Bicep {
+    BeforeAll {
+        $ScriptDirectory = Split-Path -Path $PSCommandPath -Parent
+        Copy-Item "$ScriptDirectory\supportFiles\*.bicep" -Destination TestDrive:\
+    }
+    Describe 'Build-Bicep' {
+        
+        Context 'When it works' {            
+            It 'Build a bicep file AsString' {
+                Build-Bicep -Path 'TestDrive:\workingBicep.bicep' -AsString | Should -Not -BeNullOrEmpty
+            }
+            It 'Build a bicep file AsHashtable' {
+                Build-Bicep -Path 'TestDrive:\workingBicep.bicep' -AsHashtable | Should -Not -BeNullOrEmpty
```

Comment:
Maybe we can test for an ordered hashtable instead of just a non-empty value using:
Should -BeOfType System.Collections.Specialized.OrderedDictionary